src/graph/ChcGraph.h

```diff
@@ -141,6 +165,20 @@ class ChcDirectedGraph {
     ChcDirectedGraph reverse() const;
     DirectedEdge reverseEdge(DirectedEdge const & edge, TermUtils & utils) const;
 
+    ChcDirectedGraph copy(const ChcDirectedGraph& graph)
```

Comment:
Why do you need special `copy` method? Why is not the default copy constructor good enough?

Reply:
Well, now it worked, Idk what it was before 😅 
Thanks!